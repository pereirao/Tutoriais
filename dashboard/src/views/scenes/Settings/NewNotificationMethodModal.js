// @flow

import React from 'react';
import I18n from '../../../localization';
import MBModal from '../../elements/MBModal';
import NewNotificationMethodForm from './NewNotificationMethodForm';
import style_sheet from './Settings.module.css';

type NewNotificationMethodModalProps = { hidden: boolean, close: () => void };
const NewNotificationMethodModal = ({hidden, close}: NewNotificationMethodModalProps) => {
  if (hidden) return null;
  return (
    <MBModal close={close}>
      <h3 className={style_sheet.modalHeader}>{I18n.t('ui.modal.new_notification_method.title')}</h3>
      <NewNotificationMethodForm close={close}/>
    </MBModal>
  );
};

export default NewNotificationMethodModal;
