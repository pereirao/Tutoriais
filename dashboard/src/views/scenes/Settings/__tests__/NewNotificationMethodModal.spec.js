import React from 'react';
import NewNotificationMethodModal from '../NewNotificationMethodModal';

it('renders correctly', () => {
  expect(shallow(<NewNotificationMethodModal />)).toMatchSnapshot();
});
