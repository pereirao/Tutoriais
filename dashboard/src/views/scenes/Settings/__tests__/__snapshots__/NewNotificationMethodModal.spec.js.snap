// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<MBModal
  closeable={true}
  size="medium"
>
  <h3
    className="modalHeader"
  >
    Add a New Notification Method
  </h3>
  <Connect(NewNotificationMethodForm) />
</MBModal>
`;
